rholang/src/test/scala/coop/rchain/rholang/InterpreterSpec.scala

```diff
@@ -112,17 +112,25 @@ class InterpreterSpec extends FlatSpec with Matchers {
     StoragePrinter.prettyPrint(runtime.space.store)
 
   private def success(runtime: Runtime[Task], rho: String): Task[Unit] =
-    execute(runtime, rho).map(_.swap.foreach(error => fail(s"""Execution failed for: $rho
-                                               |Cause:
-                                               |$error""".stripMargin)))
+    execute(runtime, rho).map(
```

Comment:
fwiw, the previous version (using fail) did not print `true was false` and just cut to the interesting part :) No big deal though, will not bikeshed here :)

Reply:
This behavior was preserved